satellite/console/consoleweb/consoleql/consoleerrors.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package consoleql
+
+import (
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/satellite/console"
+)
+
+// Error messages
+const (
+	internalErrDetailedMsg = "It looks like we had a problem on our end. Please try again"
+)
+
+// errConsoleInternalDetailed describes detailed error message for internal error
+var errConsoleInternalDetailed = errs.New(internalErrDetailedMsg)
+
+// HandleError returns detailed error if such error handles
+func HandleError(err error) error {
+	switch {
```

Comment:
nit: I would use an `if`.

Reply:
We have planned to create typed errors for all errors in console, so on my opinion `switch` perfectly matches here